app/build.gradle

```diff
@@ -38,6 +38,9 @@ android {
 dependencies {
 
     compile fileTree(dir: 'libs', include: ['*.jar'])
+
+
+
```

Comment:
Remnant of a dependency I previously added to attempt to test Navigation Fragment. I left the white space mistakenly.